js/src/Routes.tsx

```diff
@@ -70,6 +71,11 @@ export const RootRoutes = [
     description: "Manny's About Me Page",
     element: <MannyAboutMePage />,
   },
+  {
```

Comment:
Bryan merged in a more scalable version of routing, it may be good to take a look at that.

Reply:
okay, will do